parsers/ClinGenVariantPathogenicity/src/loadClinGenVariantPathogenicity.py

```diff
@@ -0,0 +1,142 @@
+        print(os.path.join(self.data_path, self.clingen_variant_pathogenicity_file))
+
+        with open(clingen_variant_pathogenicity_file, 'rt') as fp:
+            extractor.csv_extract(fp,
+                                  lambda line: f'CLINVARVARIANT:{line[ClinGenVariantPathogenicityCOLS.CLINVAR_VARIATION_ID.value]}',  # subject id
+                                  lambda line: f'{line[ClinGenVariantPathogenicityCOLS.MONDO_ID.value]}',  # object id
+                                  lambda line: 'is pathogenic for',  # predicate extractor
+                                  lambda line: {NODE_TYPES: SEQUENCE_VARIANT,
+                                                'Variation':line[ClinGenVariantPathogenicityCOLS.VARIATION.value],
+                                                'HGVS_Gene_Symbol':line[ClinGenVariantPathogenicityCOLS.HGNC_GENE_SYMBOL.value]},  # subject properties
+                                  lambda line: {},  # object properties
+                                  lambda line: {PRIMARY_KNOWLEDGE_SOURCE: self.provenance_id,
+                                                'Assertion' : line[ClinGenVariantPathogenicityCOLS.ASSERTION.value],
+                                                'Mode_Of_Inheritance': moi_normalizer(line[ClinGenVariantPathogenicityCOLS.MODE_OF_INHERITANCE.value],line[ClinGenVariantPathogenicityCOLS.EVIDENCE_REPO_LINK.value]),
+                                                'Applied_Evidence_Codes_Met':line[ClinGenVariantPathogenicityCOLS.APPLIED_EVIDENCE_CODES_MET.value],
+                                                'Applied_Evidence_Codes_Not_Met':line[ClinGenVariantPathogenicityCOLS.APPLIED_EVIDENCE_CODES_NOT_MET.value],
+                                                "Summary":line[ClinGenVariantPathogenicityCOLS.SUMMARY_OF_INTERPRETATION.value],
+                                                "Pubmed_Articles":line[ClinGenVariantPathogenicityCOLS.PUBMED_ARTICLES.value],
+                                                "Expert_Panel":line[ClinGenVariantPathogenicityCOLS.EXPERT_PANEL.value],
+                                                "Evidence_Repo_Link":line[ClinGenVariantPathogenicityCOLS.EVIDENCE_REPO_LINK.value],
+                                                'Guideline':line[ClinGenVariantPathogenicityCOLS.GUIDELINE.value],
+                                                'Approval_Date':line[ClinGenVariantPathogenicityCOLS.APPROVAL_DATE.value],
+                                                "Published_Date":line[ClinGenVariantPathogenicityCOLS.APPROVAL_DATE.value]},  # edge properties
+                                  comment_character='#',
+                                  delim='\t',
+                                  has_header_row=True)
+        return extractor.load_metadata
+
+# Function to normalize the mode_of_inheritance property over the edge
```

Comment:
Stylistically, this might be cleaner as a dict, something like:

```
moi_dict = {
    'Autosomal dominant inheritance': 'HP:0000006',
    'Autosomal dominant inheritance (with paternal imprinting (HP:0012274))': HPO = 'HP:0012274',
    'Autosomal dominant inheritance (mosaic)': ['HP:0000006','HP:0001442'],
    // and so on...
```

then...

```
def moi_normalizer(MOI):
    MOI = str(MOI)
    return {'label': MOI, 'HPO': moi_dict.get(MOI, '')}
```